spec/spec_helper.rb

```diff
@@ -8,13 +8,12 @@
 require "bundler/setup"
 require "sidekiq_alive"
 require "rspec-sidekiq"
-require "rack/test"
 require "debug"
+require "rack"
 
-ENV["RACK_ENV"] = "test"
 ENV["HOSTNAME"] = "test-hostname"
 
-Sidekiq.logger.level = Logger::ERROR
+Sidekiq.logger.level = Logger::FATAL
```

Comment:
Some negative scenarios will trigger error messages and this just keeps test output much neater because we don't really care about seeing those messages